x-pack/plugins/security_solution/cypress/e2e/exceptions/entry/comments.cy.ts

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ExceptionListTypeEnum, NamespaceType } from '@kbn/securitysolution-io-ts-list-types';
+import { getException, getExceptionList } from '../../../objects/exception';
+import { getNewRule } from '../../../objects/rule';
+
+import { createRule } from '../../../tasks/api_calls/rules';
+import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
+
+import {
+  esArchiverLoad,
+  esArchiverUnload,
+  esArchiverResetKibana,
+} from '../../../tasks/es_archiver';
+import { login, visitWithoutDateRange } from '../../../tasks/login';
+import {
+  addExceptionFlyoutFromViewerHeader,
+  goToEndpointExceptionsTab,
+  goToExceptionsTab,
+  openEditException,
+  openExceptionFlyoutFromEmptyViewerPrompt,
+} from '../../../tasks/rule_details';
+import {
+  addExceptionComment,
+  addExceptionConditions,
+  addExceptionFlyoutItemName,
+  clickCopyCommentToClipboard,
+  submitEditedExceptionItem,
+  submitNewExceptionItem,
+  clickOnShowComments,
+  selectOs,
+} from '../../../tasks/exceptions';
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
+import {
+  EXCEPTION_ITEM_VIEWER_CONTAINER,
+  EXCEPTION_ITEM_COMMENTS_CONTAINER_TEXT,
+  LOADING_SPINNER,
+} from '../../../screens/exceptions';
+import {
+  createEndpointExceptionList,
+  createExceptionList,
+  createExceptionListItem,
+} from '../../../tasks/api_calls/exceptions';
+import { ROLES } from '../../../../common/test';
+
+interface ResponseType {
+  body: {
+    id: string;
+    list_id: string;
+    type: ExceptionListTypeEnum;
+    namespace_type: NamespaceType;
+  };
+}
+describe('Add, copy comments in different exceptions type and validate sharing them between users', () => {
+  describe('Rule exceptions', () => {
+    beforeEach(() => {
+      esArchiverResetKibana();
```

Comment:
I think for this test we can remove this method and the `esArchiverLoad` because we aren't needing to validate that alerts were or were not generated, right?

Reply:
Please find [0e5f52f](https://github.com/elastic/kibana/pull/152757/commits/0e5f52f2e5f361024dfd130f1939d8ea285759d5)